website/blog/2021-05-20-GAAD-One-Year-Later.md

```diff
@@ -11,50 +11,18 @@ It has been one year since Facebook took the [GAAD Pledge](https://diamond.la/GA
 
 When we started there were 90 issues identified, since March when the gap analysis issues launched on Github:
 
--   11 issues have been closed by the community
+-   11 issues have been closed by the community.
 
--   19 issues were evaluated and closed by the React Native team
+-   19 issues were evaluated and closed by the React Native team.
 
--   9 Pull request were merged
+-   9 pull requests were merged.
 
--   1 Pull request has been committed to the React Native docs.
+-   1 pull request was merged into the React Native docs.
 
-We want to recognize and thank the React Native community for the significant progress towards a more accessible React Native over the past year. Contributors have advanced this work by writing pull requests and commenting on issues, along with highlighting missing information on issues and contributing input on difficult problems. Every contributor's effort has counted in making progress on improving React Native Accessibility.
-
-<!--truncate-->
```

Comment:
You have removed the `<!--truncate-->` tag from the initial version of the post, was it on purpose?

Reply:
No that was not intentional I will add it back.